dev-docs/bidders/glimpse.md

```diff
@@ -45,29 +45,15 @@ Before you start to configure Glimpse, you will need to build a `prebid.js` file
 <script async src="https://<PUBLISHER_DOMAIN>/prebid.js"></script>
 ```
 
-### Publisher Registration
-
-Coming soon.
-
-### Domain Registration
-
-Coming soon.
-
-### Ad Unit Registration
-
-Coming soon.
-
 ## Configuration
 
 ### Bid Requests
 
-Our adapter captures the following values in the `params` block of each bid request:
+Our adapter expects the following values in the `params` block of each bid request:
 
-| Name          | Scope    | Type                | Description                                                                                                                   | Example                |
-| ------------- | -------- | ------------------- | ----------------------------------------------------------------------------------------------------------------------------- | ---------------------- |
-| `placementId` | Required | string              | A unique identifier associated with a publisher and ad unit, provided by the Glimpse Publisher Portal when registering a unit | 'glimpse-placement-id' |
-| `demand`      | Optional | string              | Target demand source (defaults to 'glimpse')                                                                                  | 'xandr'                |
-| `keywords`    | Optional | Record<string, any> | Audience targeting data (applies to the single ad unit)                                                                       | { sticky: true }       |
+| Name  | Scope    | Type   | Description                                                                                         | Example                |
```

Comment:
Hi! Could you please add this "{: .table .table-bordered .table-striped }" before this table (line 53)? It'll add styling

Reply:
Thanks for the input, I've updated both tables accordingly.